src/store/messages/saga.ts

```diff
@@ -361,6 +381,7 @@ export function* receiveNewMessage(action) {
         lastMessage: message,
       })
     ),
+    call(markAllAsReadAction),
```

Comment:
I thought this would be..automatically handled..? Since we're marking as read only for active channel/conversation.

Reply:
also if we pass the channelId, wouldn't i mark the messages as read for a channel which isn't "active" (i.e receiveNewMessage on *channelX* but i am on *channelY*)

Reply:
But those sagas both have that check already, don't the?

Reply:
no they don't 👀 

bdw i am not sure how to use `takeEveryFromBus` in case i wanted the event to return a data, and pass that data to the calling function (in this case the `channelId`)
<img width="440" alt="image" src="https://github.com/zer0-os/zOS/assets/33264364/a02ba0fe-d801-4c82-9a46-3ab832078fff">

Reply:
Something like:
```
yield takeEveryFromBus(
  yield call(conversationsChannel),
  ChannelEvents.MessagesLoadedForConversation,
  (action) => markAllMessagesAsReadIfActive(action.channelId),
);
```